e2e/qwik-nx-e2e/tests/micro-frontends.spec.ts

```diff
@@ -214,10 +214,10 @@ async function runHostAndRemotes(
         const localhost = `${prefix}Local:   http://localhost:${port}/`;
         return stripped.includes(ip) || stripped.includes(localhost);
       };
-      invokedHost ||= includesInvokeMessage(hostName, 4200, false);
-      invokedRemote1 ||= includesInvokeMessage(remote1, 4201);
-      invokedRemote2 ||= includesInvokeMessage(remote2, 4202);
-      invokedRemote3 ||= !!remote3 && includesInvokeMessage(remote3, 4203);
+      invokedHost ||= includesInvokeMessage(hostName, 5173, false);
+      invokedRemote1 ||= includesInvokeMessage(remote1, 5174);
+      invokedRemote2 ||= includesInvokeMessage(remote2, 5175);
+      invokedRemote3 ||= !!remote3 && includesInvokeMessage(remote3, 5175);
```

Comment:
should also be changed here 🙈 
```suggestion
      invokedRemote3 ||= !!remote3 && includesInvokeMessage(remote3, 5176);
```